com.unity.multiplayer.mlapi/Runtime/Core/MessageBatcher.cs

```diff
@@ -15,12 +16,17 @@ public class SendStream
         {
             public string channel;
             public PooledBitStream Stream = PooledBitStream.Get();
+            public PooledBitWriter Writer;
             public bool Empty = true;
```

Comment:
so, should all public fields start with lowercase letters? :D

Reply:
Yes they should but that was part of the merge from a previous branch.
However, I am more than happy to go ahead and make the changes for all of that.

Reply:
yeah. Will fix.